sktime/utils/_testing/estimator_checks.py

```diff
@@ -129,12 +132,7 @@ def check_estimator_tags(Estimator):
     assert hasattr(Estimator, "_all_tags")
     all_tags = Estimator._all_tags()
     assert isinstance(all_tags, dict)
-    assert all(
-        [
-            isinstance(key, str) and isinstance(value, bool)
-            for key, value in all_tags.items()
-        ]
-    )
+    assert all([isinstance(key, str) for key, _ in all_tags.items()])
```

Comment:
Are the changes to the tag system necessary for this PR already?

Reply:
no, but to revert this I would have to go through multiple files at the cost of time and the risk of breaking things - it was already a hassle to split the PR into two